### PR TITLE
Update bem-core and bem-components versions

### PR DESCRIPTION
**Pull request by gela-d**

### CODE REVIEW

**Comment by gela-d** [comment]
@tavriaforever

File: bower.json [hunk 1]
```diff
@@ -4,8 +4,8 @@
   "version": "1.0.0",
   "private": true,
   "dependencies": {
-    "bem-core": "2.5.1",
-    "bem-components": "git://github.com/bem/bem-components.git#v2.0.0",
+    "bem-core": "2.6.0",
+    "bem-components": "git://github.com/bem/bem-components.git#v2.1.0",
```
Comment (tadatuta) [inline bower.json]: is there a reason not to write it like `"bem-components": "2.1.0"`?

Comment (gela-d) [inline bower.json]: @tadatuta some time ago there was some problem with it, but I forget which one :)
Now everything work correctly. Update PR.

**Comment by tadatuta** [comment]
lgtm

**Pull request merged**